pylint/extensions/_check_docs_utils.py

```diff
@@ -41,6 +41,21 @@ def returns_something(return_node):
     return not (isinstance(returns, astroid.Const) and returns.value is None)
 
 
+def inside_yields(node):
```

Comment:
Also this code grabs *all* the yields from under the node' scope, so for instance it would grab those defined in closures and whatnot. That is why I think it could be replaced with is_generator().

    >>> n = extract_node('''
    ... def test():
    ...    def best():
    ...      yield None
    ... ''')
    <FunctionDef(test) l.2 [] at 0x3041ff0>
    >>> n.nodes_of_class(astroid.Yield)
    <generator object nodes_of_class at 0x02EC8BC0>
    >>> next(_)
    <Yield() l.4 [] at 0x30430b0>
    >>>